src/backend/distributed/sql/udfs/citus_unmark_object_distributed/latest.sql

```diff
@@ -1,6 +1,7 @@
-CREATE FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int)
+CREATE FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int, checkobjectexistence boolean DEFAULT true)
     RETURNS void
     LANGUAGE C STRICT
     AS 'MODULE_PATHNAME', $$citus_unmark_object_distributed$$;
-COMMENT ON FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int)
-    IS 'remove an object address from citus.pg_dist_object once the object has been deleted';
+COMMENT ON FUNCTION pg_catalog.citus_unmark_object_distributed(classid oid, objid oid, objsubid int, checkobjectexistence boolean)
+    IS 'Removes an object from citus.pg_dist_object after deletion. If checkobjectexistence is true, object existence check performed.
+    Otherwise, object existence check is skipped.';
```

Comment:
```suggestion
    IS 'Removes an object from citus.pg_dist_object after deletion. If checkobjectexistence is true, object existence check performed.'
       'Otherwise, object existence check is skipped.';
```

Otherwise, the comment of the function would look a bit ugly, as in:
```sql
┬────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
│                                                          Description                                                           │
┼────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
│ Removes an object from citus.pg_dist_object after deletion. If checkobjectexistence is true, object existence check performed.↵│
│     Otherwise, object existence check is skipped.                                                                              │
┴────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
```